framework/docs/source/how-to-run-flower-with-deployment-engine.rst

```diff
@@ -20,115 +20,140 @@ Before you start, make sure that:
+Create a new Flower app (PyTorch), and follow the instructions show upon executing
 ``flwr new``:
 
 .. code-block:: bash
 
     $ flwr new my-project --framework PyTorch --username flower
 
-    🔨 Creating Flower project my-project...
-    🎊 Project creation successful.
+    🔨 Creating Flower App my-project...
+    🎊 Flower App creation successful.
+
+    To run your Flower App, use the following command:
 
-    Use the following command to run your project:
+            flwr run my-project
 
-          cd my-project
-          pip install -e .
-          flwr run
+    If you haven not installed all dependencies yet, follow these steps:
 
-    $ cd my-project
-    $ pip install -e .
+            cd my-project
+            pip install -e .
+            flwr run .
 
 .. note::
 
     If you decide to run the project with ``flwr run .``, the Simulation Engine will be
     used. Continue to Step 2 to know how to instead use the Deployment Engine.
 
-Step 2: Start the SuperLink
----------------------------
+.. tip::
 
-In a new terminal, start the SuperLink process in insecure mode:
+    Feel free to inspect the code using your favorite code editor before proceeding.
+    Just open the ``my-project`` that was automatically created via ``flwr new``. If you
+    would like to get an overview of the code that was generated, take a look at the
+    :doc:`tutorial-quickstart-pytorch` tutorial.
+
+Step 2: Launch the SuperLink
+----------------------------
+
+In a new terminal, activate your environment and start the ``SuperLink`` process in
+insecure mode:
 
 .. code-block:: bash
 
     $ flower-superlink --insecure
 
 .. dropdown:: Understand the command
 
-    * ``flower-superlink``: Name of the SuperLink binary.
+    * ``flower-superlink``: Name of the ``SuperLink`` binary.
     * | ``--insecure``: This flag tells the SuperLink to operate in an insecure mode, allowing
-      | unencrypted communication.
+      | unencrypted communication. Refer to the :doc:`how-to-enable-tls-connections` guide to learn how to run your ``SuperLink`` with TLS.
 
-Step 3: Start the SuperNodes
-----------------------------
+Step 3: Launch the SuperNodes
+-----------------------------
 
+In this step, you will launch two ``SuperNodes`` and connect them to the ``SuperLink``.
 You will need two terminals for this step.
 
-1. **Terminal 1** Start the first SuperNode:
+.. note::
+
+    Note that the values passed via the ``--node-config`` argument are specific to the
+    behaviour of the ``ClientApp``. If you inspect the code generated in the first step
+    via ``flwr new``, you'd see that the ``ClientApp`` is expecting a certian set of
+    key-value pairs to be present in order to partition and load some data. Typically,
+    your ``ClientApp`` wouldn't partition a dataset, instead it would access the data
+    directly available. In such cases you would write your ``ClientApp`` and make it
+    recieve, for example, the path to a directory of images.
```

Comment:
```suggestion
    Note that the values passed via the ``--node-config`` argument are specific to the
    behaviour of the ``ClientApp``. If you inspect the code generated in the first step
    via ``flwr new``, you'd see that the ``ClientApp`` is expecting a certian set of
    key-value pairs to be present in order to partition and load some data. Typically,
    your ``ClientApp`` wouldn't partition a dataset, instead it would access the data
    directly available. In such cases you would write your ``ClientApp`` and make it
    receive, for example, the path to a directory of images.
```